bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/handler/capability/RefreshCapability.java

```diff
@@ -82,38 +84,36 @@ private boolean probing() {
     private void proceedWithUpdate() {
         handler.proceedWithUpdate();
         long delay;
-        if (!ThingStatus.ONLINE.equals(handler.getThing().getStatus())) {
-            logger.debug("Module is not ONLINE; special refresh interval is used");
-            delay = OFFLINE_INTERVAL.toSeconds();
-            if (probing()) {
-                dataTimeStamp0 = Instant.MIN;
-            }
+        if (!ACTIVE_STATUSES.contains(thing.getStatus())) {
```

Comment:
I'm a bit confused about this logic. Does it mean "if we are offline, don't try to get online again"? It seems a bit backwards?

Reply:
In my understanding, trying to get it back again is what produces the ghost handlers. Only the API bridge should start to resume, and then cascade child things but I must admit that I start to get lost... :-/

Reply:
I'm resolving this comment since the change was reverted.